### PR TITLE
cleanup/refactor/fix app control webpack config 

### PR DESCRIPTION
**Pull request by jchip**

### CODE REVIEW

File: samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { loadSubApp } from "subapp-web";
+import PropTypes from "prop-types";
+
+let data;
+
+class Data1 extends React.Component {
+  constructor() {
+    super();
```
Comment (christianlent) [inline samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js]: Just curious, why are you overriding the constructor in these demos? Is it intentionally clearing out the construction props? If not, why not pass the props to `super`?

File: samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  render(props) {
```
Comment (christianlent) [inline samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js]: And similarly, why does this take props?

File: samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js (hunk 1 above)
Comment (jchip) [reply samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js]: good catch. :) thanks.

Comment (jchip) [reply samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js]: this change actually seem to got mixed up with another PR.  ahh... trying to juggle 5 PRs with git. :)

**Pull request merged**